src/Orleans.Core/Statistics/StatisticsCollector.cs

```diff
@@ -1,108 +0,0 @@
-using Orleans.Runtime.Configuration;
-
-namespace Orleans.Runtime
-{
-    internal static class StatisticsCollector
```

Comment:
this change looks good to me. the only thing I would suggest is since we splitted `StatisticsCollector ` and now each StatisticGroupe handle the bool on should the system collect their metrics or not in their own logic. This is consistent with the direction we are going with. But we also lost this single place we can look up on what does `StatisticsLevel ` really mean. Set it to a specific level would enable which metrics.  So I suggest put this info in `statisticsOptions.CollectionLevel ` xml comment

Reply:
Sounds good to me, I'll come up with a better solution to keep some of the semantics localized